packages/angular-query-experimental/src/__tests__/zonejs-adapter.test.ts

```
@@ -0,0 +1,96 @@
+  Component,
+  provideZoneChangeDetection,
+} from '@angular/core'
+import { TestBed } from '@angular/core/testing'
+import { afterEach, beforeEach, describe, expect, it, vi } from 'vitest'
+import {
+  QueryClient,
+  injectMutation,
+  injectQuery,
+  provideTanStackQuery,
+} from '..'
+import { sleep } from '@tanstack/query-test-utils'
```

ESLint

[error] `@tanstack/query-test-utils` import should occur before import of `..`

(import/order)